common/time.hpp

```diff
@@ -15,4 +15,12 @@ static inline Time timestamp() {
     return (Time)time.tv_sec * 1000000 + (Time)time.tv_usec;
 }
 
+static inline RJ::Time SecsToTimestamp(double secs){
```

Comment:
It might also be nice to create an overload one that takes in an u_int which doesn't lose precision to double multiplication.